app/cdap/components/WranglerGrid/AddTransformationPanel/utils.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { DATATYPE_OPTIONS } from 'components/WranglerGrid/NestedMenu/menuOptions/datatypeOptions';
+
+export const getDirective = (functionName: string, selectedColumnName: string) => {
+  if (DATATYPE_OPTIONS.some((eachOption) => eachOption.value === functionName)) {
+    return `set-type :${selectedColumnName} ${functionName}`;
+  } else {
+    return null;
```

Comment:
What's the purpose of returning `null` here? If this is an error state, why not throw an exception?

Reply:
We don't require returning null, here it was used just if transformation name doesn't exist, but there will never be any case when the transformation name does not match it will always return a expected value. So removed else block.